docs/content/foundations/typography/text-styles.mdx

```diff
@@ -4,145 +4,262 @@ page_title: Text Styles
+              hex_value: '#007bbd'
+            - class_name: 'underline-primary-vivid'
+              hex_value: '#067cbc'
+            - class_name: 'underline-primary-dark'
+              hex_value: '#004971'
+            - class_name: 'underline-primary-darker'
+              hex_value: '#00314b'
+            - class_name: 'underline-secondary-lighter'
+              hex_value: '#beebee'
+            - class_name: 'underline-secondary-light'
+              hex_value: '#4bbfc6'
+            - class_name: 'underline-secondary'
+              hex_value: '#298085'
+            - class_name: 'underline-secondary-vivid'
+              hex_value: '#338084'
+            - class_name: 'underline-secondary-dark'
+              hex_value: '#1e4c4f'
+            - class_name: 'underline-secondary-darker'
+              hex_value: '#17373a'
+            - class_name: 'underline-accent-cool-lighter'
+              hex_value: '#d7e5f4'
+            - class_name: 'underline-accent-cool-light'
+              hex_value: '#92a9c8'
+            - class_name: 'underline-accent-cool'
+              hex_value: '#5478ab'
+            - class_name: 'underline-accent-cool-dark'
+              hex_value: '#284976'
+            - class_name: 'underline-accent-cool-darker'
+              hex_value: '#06162d'
+            - class_name: 'underline-accent-warm-lighter'
+              hex_value: '#fdf2bf'
+            - class_name: 'underline-accent-warm-light'
+              hex_value: '#fee685'
+            - class_name: 'underline-accent-warm'
+              hex_value: '#face00'
+            - class_name: 'underline-accent-warm-dark'
+              hex_value: '#ddaa01'
+            - class_name: 'underline-accent-warm-darker'
+              hex_value: '#b38c00'
+            - class_name: 'underline-emergency'
+              hex_value: '#b60d43'
+            - class_name: 'underline-emergency-dark'
+              hex_value: '#700824'
+        - heading: Grayscale Palette
+          classes:
+            - class_name: 'underline-gray-5'
+              hex_value: '#f0f0f0'
+            - class_name: 'underline-gray-10'
+              hex_value: '#e6e6e6'
+            - class_name: 'underline-gray-30'
+              hex_value: '#adadad'
+            - class_name: 'underline-gray-50'
+              hex_value: '#757575'
+            - class_name: 'underline-gray-70'
+              hex_value: '#454545'
+            - class_name: 'underline-gray-90'
+              hex_value: '#171717'
+        - heading: Basic Palette
+          classes:
+            - class_name: 'underline-teal'
+              hex_value: '#298085'
+            - class_name: 'underline-cerulean'
+              hex_value: '#007bbd'
+            - class_name: 'underline-cranberry'
+              hex_value: '#bb0e3d'
+            - class_name: 'underline-navy'
+              hex_value: '#14315c'
+            - class_name: 'underline-golden'
+              hex_value: '#947100'
+  - name: Uppercase and lowercase
```

Comment:
Not a note but `THE CELEBRATED JUMPING FROG OF CALAVERAS COUNTY` made me laugh because it looks like yelling about a frog 😂